src/couch_mrview_http.erl

```diff
@@ -576,6 +576,9 @@ check_view_etag(Sig, Acc0, Req) ->
 
 
 parse_json(V) when is_list(V) ->
-    ?JSON_DECODE(V);
+    case lists:all(fun is_binary/1, V) of
```

Comment:
this feels like a nasty hack to me, with potential for unintended side-effects. The original multi-query feature in the cloudant contribution did not suffer this problem.